code/modules/power/tesla/coil.dm

```diff
@@ -56,6 +61,12 @@
 	else
 		..()
 
+/obj/machinery/power/tesla_coil/attack_hand(mob/user)
+	if(user.a_intent == INTENT_GRAB)
+		return user_buckle_mob(user.pulling, user, check_loc = FALSE)
```

Comment:
User.pulling? Maybe user.grabbed or something instead?

Reply:
there is no user.grabbed you have to kind of do something like 
if(user.a_intent == INTENT_GRAB)
		var/obj/item/weapon/grab/G = user.get_active_hand()
		if(isliving(G.affecting))
			return user_buckle_mob(G.affecting, user, check_loc = FALSE)
whitch does work so i will have to do it hte other way @Kyep so it works.

Reply:
I guess my thing is... how do you normally buckle mobs to things? Personally, the only way I ever do it is to move them on top of the thing, then click-drag them to the thing. We can't do this here as these are density = 1 objects. Ok. Grab still makes more sense than pulling, I think. Whatever method used, it should probably be similar to the meat spike that people already use - all objects which are solid, but which you can put people onto, should probably use the same method. If they all have their own special ways to put someone on them, there is no way players are going to take the time to learn each technique.

Reply:
doesen't work rather. not does work.

Reply:
@Kyep i went off the meakspike and it still didn't work unless i just did not do the code right somehow.

Reply:
kitchen_spike.dm #43 and #56 have a fewer-line way to check for grabs. But yours may be neater, so.